docs/configuration/application_ports.md

```diff
@@ -43,6 +43,7 @@ By default, applications can be found on the ports listed below.
 | Portainer       | 9000   |           |
 | pyload          | 8000   |           |
 | Radarr          | 7878   |           |
+| Sabnzbd         | 18080  | HTTP      |
```

Comment:
You should also disclose the HTTPS port since this one is also redirected

Reply:
Got it.
-Dale